### ISSUE TITLE
add data-imagesrc to display images

### ISSUE THREAD

**Issue by Webdinge** [hernansartorio/jquery-nice-select#63]
Hi,

Great lib!!
How can we add the possibility to show images in the newly created dropdown? I was thinking about using something like `data-imgsrc=some-image.png` which will be added as image in the dropdown.

Any pointers or help appreciated :)

**Comment by huebacca** [comment]
I agree this would be great! I'm not sure how the rendering would go in the DOM with an image but it's certainly something I'd be interested in looking into.

**Comment by ershagen** [comment]
Hi Webdinge :) I did this with data-image. See: https://github.com/hernansartorio/jquery-nice-select/pull/70

**Comment by letownia** [comment]
I'll add that the master version doesn't include this commit,, and you have to checkout the actual commit: https://github.com/hernansartorio/jquery-nice-select/pull/70

**Comment by mshshuvooo** [comment]
You can check this https://github.com/hernansartorio/jquery-nice-select/blob/ef1c248fc8f190470c7f15b0f3e197c49211d80b/js/jquery.nice-select.js
